source/game/p_client.cpp

```diff
@@ -798,6 +800,30 @@ void ClientBegin( edict_t *ent )
 	G_Gametype_ScoreEvent( client, "enterGame", NULL );
 }
 
+void ClientAuth ( edict_t *ent, uint64_t steamid )
+{
+	gclient_t *client = ent->r.client;
+	client->steamid = steamid;
+	client->authenticated = true;
+
+	if ( g_permanent_operators->string[0] ){
+		char *pch = strtok(g_permanent_operators->string, ",");
+		while (pch != NULL){
+			uint64_t testid = atol(pch);
```

Comment:
Some people recommend `strtol` instead of `atol`:
https://stackoverflow.com/questions/3792663/atol-v-s-strtol